packages/cva/src/index.ts

```diff
@@ -74,7 +82,7 @@ type CVAVariantShape = Record<string, Record<string, ClassValue>>;
 type CVAVariantSchema<V extends CVAVariantShape> = {
   [Variant in keyof V]?: StringToBoolean<keyof V[Variant]> | undefined;
 };
-type CVAClassProp =
+export type CVAClassProp =
```

Comment:
Any reason for adding the `export`?

Reply:
Hey @joe-bell thanks for the quick review, and sorry for the slow response, I was on the road after I submitted this, until today.

I must have added this when I was tinkering around, in order to use this type in a scratch test file; I'll remove it.

I also realized after the fact that I could possibly add some type-testing coverage, since you have vitest set up here; and that I hadn't written my commit message with the conventional format. Thoughts?